web/src/helpers/fetcher.ts

```diff
@@ -32,3 +32,7 @@ export const fetcher = async (input: RequestInfo, init: RequestInit) => {
     throw new FetcherError([(error as Error).toString()], 500, {});
   }
 };
+
+export function fetchWithCreds(input: RequestInfo, init: RequestInit) {
```

Comment:
```suggestion
export function fetcherWithCreds(input: RequestInfo, init: RequestInit) {
```

Считаю, что нужно оставить основной нейм.

Reply:
а почему у тебя функция не является глаголом, а существительным?

Reply:
Я на счет этом не задумывался. Моя цель была выбрать уникальное и понятное имя. 

Можешь предложить как это всё назвать, чтобы не было повторения текущего кода:
- fetcher
- FetcherError
- useFetcher

Reply:
И давай использовать один стиль определения функций.

Я всегда использую стрелочный вариант const fn = () => {}

Reply:
Ясно, предлагаю переименовать на:
- fetchData
- FetchError
- useFetch

после того как свои функции я удалю

Reply:
ну это вкусовщина же, чем плох function?